README.md

```diff
@@ -22,7 +21,8 @@ console.log(results);
 ```
 
 ```js
-// Your answer...
+var result = AuthorModel.findOne({name: "Bob"});
+console.log(result);
 ```
```

Comment:
This code is not using a callback function to process the results. Mongoose DB operations (find, save, etc) are all async, which means we need to pass a callback function to handle the results when they come back. Example below:

var results = AuthorModel.find({name: "Bob"}, function(results){
  console.log(results);
});